src/System.DirectoryServices.AccountManagement/src/PinvokeAnalyzerExceptionList.analyzerdata

```diff
@@ -1,4 +1,23 @@
-<!-- Temporary workaround to get build to work. -->
-credui.dll!CredUIParseUserNameW
-advapi32.dll!LsaNtStatusToWinError
 activeds.dll!ADsOpenObject
```

Comment:
Why do we need to have these baselines even on assemblies like System.DirectoryServices.* that have near zero chance of ever being in UWP?